projects/js-packages/publicize-components/jest-globals.js

```diff
@@ -10,3 +10,11 @@ if ( ! window.matchMedia ) {
 		dispatchEvent: jest.fn(),
 	} );
 }
+
```

Comment:
Do we need to leave this in?

Reply:
Do you mean the initial state? If so, then yes, all the tests started failing without it. It seems like the connection is being used differently. We might be able to mock something else, but I noticed that Backup and a couple of other places had taken this approach.